config.js

```diff
@@ -41,7 +41,7 @@ module.exports = {
     pollingDelay: process.env.POLLING_DELAY_SANDBOX || pollingDelay,
     useNewPDBridge: process.env.USE_NEW_PD_BRIDGE_SANDBOX || false,
     pdBridgeUrl: process.env.PD_BRIDGE_URL_SANDBOX || null,
-    redisCacheUrl: process.env.REDIS_CACHE_URL_SANDBOX,
+    redisCacheUrl: process.env.REDIS_URL,
```

Comment:
should this be sandbox and the other prod ?

Reply:
Since the heroku redis is managed, they set the env var REDIS_URL to the url of the instance and it changes if they ever swap it out, which happens from time to time.